grpc/server/src/main/java/io/helidon/grpc/server/SSLContextBuilder.java

```diff
@@ -58,7 +58,7 @@ private SSLContextBuilder() {
      * @param privateKeyConfig the required private key configuration parameter
      * @return this builder
      */
-    public static SSLContextBuilder create(KeyConfig privateKeyConfig) {
+    public static SSLContextBuilder create(Keys privateKeyConfig) {
```

Comment:
I previously said this, but I think we should conventionalize Config suffix...

Reply:
Config suffix should be for cases where we use the prototype to configure another object. In this case the prototype is the "final" product.